setup.cake

```diff
@@ -0,0 +1,106 @@
+#load nuget:https://www.myget.org/F/cake-contrib/api/v2?package=Cake.Recipe&prerelease
+
+Environment.SetVariableNames();
+
+BuildParameters.SetParameters(context: Context,
+                            buildSystem: BuildSystem,
+                            sourceDirectoryPath: "./src",
+                            title: "Cake.Bakery",
+                            repositoryOwner: "cake-build",
+                            repositoryName: "bakery",
+                            appVeyorAccountName: "cakebuild",
+                            shouldRunDotNetCorePack: true,

```

Comment:
NuSpec is just for the Bakery tool (Full CLR and CoreCLR separately). Libs will be packed without NuSpec.

Will override default task to use ours.